simulariumio/data_objects/display_data.py

```diff
@@ -62,10 +62,18 @@ def __init__(
         # it's easy to test whether the user has specified it
         self.radius = radius
         if not isinstance(display_type, DISPLAY_TYPE):
-            raise DataError(
-                f"In {name}'s DisplayData, display_type is {type(display_type)}"
-                " instead of DISPLAY_TYPE"
-            )
+            try:
+                display_type = DISPLAY_TYPE[display_type]
+            except KeyError:
+                raise DataError(
+                    f"In {name}'s DisplayData, {display_type} is not a display type, "
+                    'try "SPHERE" or see DISPLAY_TYPE documentation for other options.'
+                )
+            if not isinstance(display_type, DISPLAY_TYPE):
+                raise DataError(
+                    f"In {name}'s DisplayData, display_type is {type(display_type)}"
+                    " instead of DISPLAY_TYPE"
+                )
```

Comment:
What's an example situation where you'd NOT get a `KeyError` in line 67, proceed to line 72, and run into the error in line 73 because it's still not a valid display type?

Reply:
yeah it shouldn't be possible to get to line 72, removed that